yarn/src/main/scala/org/apache/spark/deploy/yarn/ClientArguments.scala

```diff
@@ -61,11 +61,15 @@ private[spark] class ClientArguments(args: Array[String], sparkConf: SparkConf)
 
   // Additional memory to allocate to containers
   val amMemoryOverheadConf = if (isClusterMode) driverMemOverheadKey else amMemOverheadKey
-  val amMemoryOverhead = sparkConf.getInt(amMemoryOverheadConf,
-    math.max((MEMORY_OVERHEAD_FACTOR * amMemory).toInt, MEMORY_OVERHEAD_MIN))
-
-  val executorMemoryOverhead = sparkConf.getInt("spark.yarn.executor.memoryOverhead",
-    math.max((MEMORY_OVERHEAD_FACTOR * executorMemory).toInt, MEMORY_OVERHEAD_MIN))
+  val amMemoryOverheadDefault = math.max((MEMORY_OVERHEAD_FACTOR * executorMemory).toInt,
+                                         MEMORY_OVERHEAD_MIN)
```

Comment:
Indentation is wrong. Please look at the code you're removing for how this should be indented.